synapse/cortex.py

```diff
@@ -804,21 +819,31 @@ def _getSynIngestNodes(self, item):
                 pnodes.append(obj)
         return pnodes
 
-    def _initCoreDir(self):
-
-        # each cortex has a default write layer...
-        s_common.gendir(self.dirn, 'layers', 'default')
+    def _migrateOldDefaultLayer(self):
```

Comment:
lets add a `#pragma no:cover` here and a fixme to remove it prior to 010 release.

Reply:
ok